eventuals/event-loop.h

```diff
@@ -422,6 +422,11 @@ class EventLoop final : public Scheduler {
             Errors,
             std::tuple<std::runtime_error>>;
 
+        template <typename Downstream>
+        static constexpr bool CanCompose = Downstream::ExpectsValue;
+
+        using Expects = SingleValue;
```

Comment:
Timer's Start function takes void as argument. `using Expects = Nothing`?

Reply:
`void` is a "single value"!